### DISCUSSION TITLE
Wish it allowed larger VMs in individually-owned spaces

### DISCUSSION THREAD

**Discussion by sourcefrog** [community/community#21324] [Codespaces] [answered]
"Shut up and take my money" etc.

I am using this as a solo developer on repos already owned by my personal account. I just want my CPU-intensive Rust tests to run faster.

I realize this behavior is by design (https://github.com/community/community/discussions/4280) but I wish you would change it.

Also, I wish I did not have to create an organization just so I can use Codespaces by myself. I imagine you're concerned about abuse by cryptocurrency mining but this seems like a kinda indirect way to deal with it.

**Comment by samruddhikhandale** [comment, marked as answer]
Codespaces for personal repositories is currently in beta, we’ll share updates on what’s coming in the near future.

Read more on our [Codespaces page](https://github.com/features/codespaces) and in [docs](https://docs.github.com/codespaces).

**Reply by stixxx123** [reply]
2,2 trillion
